### ISSUE TITLE
[BUG] Spamassasin cron failed with invalid regexp

### ISSUE THREAD

**Issue by marcinkunert** [docker-mailserver/docker-mailserver#2417]
### Miscellaneous first checks

- [X] I checked that all ports are open and not blocked by my ISP / hosting provider.
- [X] I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.

### Affected Component(s)

Spamassasin cron failed

### What happened and when does this occur?

I've received an error message from the daily cron

Topic: `Cron <root@mail> test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )`

Messsage:
```
/etc/cron.daily/spamassassin:
config: invalid regexp for __URI_TRY_3LD 'm,^https?://(?:try(?!r\.codeschool)|start|get(?!\.adobe)|save|check(?!out)|act|compare|join|learn(?!ing)|request|visit(?!or|\.vermont)|my(?!sub|turbotax|news\.apple|a\.godaddy|account|support|build|blob)\w)[^.]*\.[^/]+\.(?<!list-manage\.)(?:com|net)\b,i': Variable length lookbehind is experimental in regex; marked by <-- HERE in m/(?i)^https?://(?:try(?!r\.codeschool)|start|get(?!\.adobe)|save|check(?!out)|act|compare|join|learn(?!ing)|request|visit(?!or|\.vermont)|my(?!sub|turbotax|news\.apple|a\.godaddy|account|support|build|blob)\w)[^.]*\.[^/]+\.(?<!list-manage\.)(?:com|net)\b <-- HERE /

channel 'updates.spamassassin.org': lint check of update failed, channel failed
sa-update failed for unknown reasons
```

### What did you expect to happen?

```Markdown
THe cron should run without failing.
```

### How do we replicate the issue?

```Markdown
No idea, I've recently upgraded from v9.1 to v10.4.0, but removed most of the custom config. I only got `config/postfix-main.cf`

``
postscreen_dnsbl_sites =
smtpd_recipient_restrictions = permit_sasl_authenticated, permit_mynetworks, reject_unauth_destination, check_policy_service unix:private/policyd-spf, reject_unauth_pipelining, reject_invalid_helo_hostname, reject_non_fqdn_helo_hostname, reject_unknown_recipient_domain
#smtpd_recipient_restrictions = permit_sasl_authenticated, permit_mynetworks, reject_unauth_destination, check_policy_service unix:private/policyd-spf, reject_unauth_pipelining, reject_invalid_helo_hostname, reject_non_fqdn_helo_hostname, reject_unknown_recipient_domain, reject_rbl_client zen.spamhaus.org

# (default 5d)
maximal_queue_lifetime = 3d
# max time in deferred
bounce_queue_lifetime = 2d
# newline to make sure it reads the last line
```

### DMS version

v10.4.0

### What operating system is DMS running on?

Linux

### What instruction set architecture is DMS running on?

x86_64 / AMD64

### What container orchestration tool are you using?

Docker Compose

### docker-compose.yml

```yaml
version: '3.8'

services:
  mail:
    image: docker.io/mailserver/docker-mailserver:10.4.0
    hostname: ${HOSTNAME}
    domainname: ${DOMAINNAME}
    container_name: ${CONTAINER_NAME}
    ports:
      - "25:25"
      - "143:143"
      - "587:587"
      - "993:993"
    volumes:
      - maildata:/var/mail
      - mailstate:/var/mail-state
      - maillogs:/var/log/mail
      - ./config/:/tmp/docker-mailserver/
      - /etc/letsencrypt/:/etc/letsencrypt/
      - /etc/localtime:/etc/localtime:ro
    env_file:
      - mailserver.env
    cap_add:
      - NET_ADMIN
      - SYS_PTRACE
    restart: always

volumes:
  maildata:
  mailstate:
  maillogs:
```

### Relevant log output

```shell
I haven't found anything related with this in logs.
```

### Other relevant information

```Markdown
I've noticed a similar issue #2407, but the cause there was a custom Spamassasin config, which I don't have.
```

### What level of experience do you have with Docker and mail servers?

- [ ] I am inexperienced with docker
- [ ] I am inexperienced with mail servers
- [ ] I am uncomfortable with the CLI

### Code of conduct

- [X] I have read this project's [Code of Conduct](https://github.com/docker-mailserver/docker-mailserver/blob/master/CODE_OF_CONDUCT.md) and I agree
- [X] I have read the [README](https://github.com/docker-mailserver/docker-mailserver/blob/master/README.md) and the [documentation](https://docker-mailserver.github.io/docker-mailserver/edge/) and I searched the [issue tracker](https://github.com/docker-mailserver/docker-mailserver/issues?q=is%3Aissue) but could not find a solution

**Comment by ontheair81** [comment]
Just to mention that you are not alone: I received the same error message today, for the first time.

Running `edge` since a few days (as I wanted to benefit from the [fixed check-for-changes.sh-issue](https://github.com/docker-mailserver/docker-mailserver/issues/2348)). No custom config anymore, as I am using now ENABLE_DNSBL to disable DNS blocks, instead of the [manual workaround](https://github.com/docker-mailserver/docker-mailserver/issues/1927#issuecomment-824940913).

**Comment by casperklein** [comment]
I saw this error as well this morning.

> Maybe, just maybe, this issue will resolve "itself" when a new :edge image is build

I don't see, how this error is in any way related to the edge build.

> channel 'updates.spamassassin.org': lint check of update failed, channel failed

The message clearly states, that something is messed up upstream with the provided pattern update.

I suggest to relax and wait, until this is fixed upstream 😎

**Comment by georglauterbach** [comment]
> Edit: This is a [known issue](https://spamassassin.markmail.org/search/?q=#query:+page:1+mid:ydcrwxmymidcwxxy+state:results) and not related to DMS.

Nice. I will leave this pinned until the issue is resolved :)

**Comment by casperklein** [comment]
The issue has been fixed. Verify with:

`dig txt 4.4.3.updates.spamassassin.org +short` should now return `TXT "1898205"`, before it was `TXT "1898171"`.

To update the SA pattern you can wait one day or trigger the update manually with:

`docker exec -it mailserver bash -c /etc/cron.daily/spamassassin`
